dist-material/log4j2.xml

```diff
@@ -32,12 +32,22 @@
             </Policies>
             <DefaultRolloverStrategy max="30"/>
         </RollingFile>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout>
+                <pattern>%m%n</pattern>
+            </PatternLayout>
+            <MarkerFilter marker="Server started" onMatch="ACCEPT" onMismatch="DENY"/>
```

Comment:
What do you think if we rename the marker to something like `StandardOut` or `Console`?

Reply:
It should be possible. Let me try.